wls-gui-wahllokalsystem/src/stores/userStore.ts

```diff
@@ -8,7 +8,26 @@ import { WahlbezirksArtEnum } from "@/types/wahlbezirksArtEnum.ts";
 const { getUser } = useUserService();
 
 export const useUserStore = defineStore("user", () => {
-  const user = ref<User | null>(null);
+  const defaultUser = {
+    authorities: new Set<string>(),
+    email: "",
+    pin: "",
+    userEnabled: false,
+    username: "",
+    wahlMetaData: [
+      {
+        wahlbezirkID: "",
+        wahlnummer: "",
+        wahlID: "",
+      },
+    ],
+    wahlbezirkID: "",
+    wahlbezirkNummer: "",
+    wahlbezirksArt: WahlbezirksArtEnum.UWB,
+    wahltag: "",
+    wahltagID: "",
+  };
+  const user = ref<User>(defaultUser);
```

Comment:
warum braucht es hier nochmal einen default user, wenn in `User.ts` schon default werte hinterlegt sind? wen du das umsetzt muss der default für die wahlbezirksart in `USer.ts` angepasst werden
```suggestion
  const user = ref<User>(new User());
```